### PR TITLE
fix nimbus multiple keys import

### PR DESCRIPTION
**Pull request by xenowits**
Fixes issue with importing multiple keystores in nimbus. Thanks @dB2510 for pointing this out.

ticket: https://github.com/ObolNetwork/charon-distributed-validator-cluster/issues/67
category: bug

### CODE REVIEW

**Comment by xenowits** [comment]
Screenshot 2023-02-15 at 12 00 04
Screenshot 2023-02-15 at 12 00 49

**Comment by xenowits** [comment]
Logs from nimbus VC:
```
...
charon-distributed-validator-cluster-vc5-nimbus-1  | DBG 2023-02-15 06:26:16.039+00:00 Sending attestation                        attestation="(aggregation_bits: 0b00000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000, data: (slot: 4994531, index: 10, beacon_block_root: \"c9cd8ae2\", source: \"156078:a41fc4d7\", target: \"156079:30b17c30\"), signature: \"a9d66f28\")" validator=aef3d97c validator_index=356280 delay=40ms879us421ns service=attestation_service
charon-distributed-validator-cluster-vc5-nimbus-1  | NOT 2023-02-15 06:26:16.062+00:00 Attestation published                      attestation="(aggregation_bits: 0b00000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000, data: (slot: 4994531, index: 10, beacon_block_root: \"c9cd8ae2\", source: \"156078:a41fc4d7\", target: \"156079:30b17c30\"), signature: \"a9d66f28\")" validator=aef3d97c validator_index=356280 delay=62ms818us171ns service=attestation_service
charon-distributed-validator-cluster-vc5-nimbus-1  | DBG 2023-02-15 06:26:16.063+00:00 Attestation statistics                     total=1 succeed=1 failed_to_deliver=0 not_accepted=0 delay=63ms90us129ns slot=4994531 committee_index=10 duties_count=1 service=attestation_service
...
```

**Comment by xenowits** [comment]
Logs from charon (note that all 6 partial signatures are received):
```
charon-distributed-validator-cluster-node5-1  | 07:14:16.129 DEBG vapi       Partial signed data stored               {"count": 1, "pubkey": "990_14a", "duty": "4994771/attester", "vapi_endpoint": "submit_attestations"}
charon-distributed-validator-cluster-node5-1  | 07:14:16.158 DEBG parsigex   Partial signed data stored               {"count": 2, "pubkey": "990_14a", "duty": "4994771/attester", "peer": "lively-dates"}
charon-distributed-validator-cluster-node5-1  | 07:14:16.199 DEBG parsigex   Partial signed data stored               {"count": 3, "pubkey": "990_14a", "duty": "4994771/attester", "peer": "shiny-skin"}
charon-distributed-validator-cluster-node5-1  | 07:14:16.202 DEBG parsigex   Partial signed data stored               {"count": 4, "pubkey": "990_14a", "duty": "4994771/attester", "peer": "prickly-taxi"}
charon-distributed-validator-cluster-node5-1  | 07:14:16.254 DEBG sigagg     Threshold aggregated partial signatures  {"duty": "4994771/attester", "peer": "prickly-taxi"}
charon-distributed-validator-cluster-node5-1  | 07:14:16.262 INFO bcast      Successfully submitted attestation to beacon node {"delay": "4.262146004s", "pubkey": "990_14a", "duty": "4994771/attester", "peer": "prickly-taxi"}
charon-distributed-validator-cluster-node5-1  | 07:14:16.289 DEBG parsigex   Partial signed data stored               {"count": 5, "pubkey": "990_14a", "duty": "4994771/attester", "peer": "enthusiastic-future"}
charon-distributed-validator-cluster-node5-1  | 07:14:16.312 DEBG parsigex   Partial signed data stored               {"count": 6, "pubkey": "990_14a", "duty": "4994771/attester", "peer": "long-money"}
```

File: nimbus/run.sh [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+
 # Refer: https://nimbus.guide/keys.html
 # Running a nimbus VC involves two steps which need to run in order:
 # 1. Importing the validator keys
 # 2. And then actually running the VC
+tmpkeys="/home/validator_keys/tmpkeys"
+mkdir -p ${tmpkeys}
+
 for f in /home/validator_keys/keystore-*.json; do
   echo "Importing key ${f}"
```
Comment (dB2510) [inline nimbus/run.sh:14]: can we keep this after import? like `echo "Imported key ${f}"`

**Review by dB2510 — APPROVED** [review]

File: nimbus/run.sh (hunk 1 above)
Comment (xenowits) [reply nimbus/run.sh:14]: We can but for consistency's sake (see lighthouse/run.sh), let's keep it this way

**Comment by xenowits** [comment]
Note that this won't work with v0.13.0 or older. Please use latest release (v0.14.0) or latest main

**Pull request merged**